### PR TITLE
Docs: Move `-params-file` config from CLI to config

### PR DESCRIPTION
**Pull request by christopher-hakkaart**
- Added params file section to the config page
- Reduced CLI documentation (not the place to describe JSON and YAML)
    - Added link to config page in the example

cc @FriederikeHanssen

@netlify /config#params-file

### CODE REVIEW

File: docs/config.md [hunk 1]
```diff
@@ -331,3 +331,37 @@ workflow.onError = {
...
+nextflow run main.nf -params-file pipeline_params.yml
+```
+
+The `-params-file` option loads parameters for your Nextflow pipeline from a JSON or YAML file. Parameters defined in the file are equivalent to specifying them directly on the command line. For example:
+
+```bash
+nextflow run main.nf --alpha 1 --beta foo
+```
+
+The above parameters can be represented in YAML format:
```
Comment (ewels) [inline docs/config.md:351]: Feels like this doesn't quite follow on / flow from the text above.
```suggestion
would be equivalent to the following in YAML format:
```

**Review by ewels — APPROVED** [review]
LGTM, thanks!

File: docs/config.md (hunk 1 above)
Comment (christopher-hakkaart) [reply docs/config.md]: It was feeling uneasy for me too. New flow proposed.

**Review by bentsherman — CHANGES_REQUESTED** [review]
Params file should be described [here](https://nextflow.io/docs/latest/cli.html#pipeline-parameters) (on the Command line page) rather than the Config page. You can add a backlink to the Command line page from [here](https://nextflow.io/docs/latest/config.html#parameters)

**Review by bentsherman — APPROVED** [review]

**Review by pditommaso — APPROVED** [review]

**Pull request merged**